pom.xml

```diff
@@ -1182,30 +1182,6 @@
                 <artifactId>aspectjweaver</artifactId>
                 <version>1.8.13</version>
             </dependency>
-            <dependency>
-                <groupId>eu.medsea.mimeutil</groupId>
-                <artifactId>mime-util</artifactId>
-                <version>2.1.3</version>
-                <exclusions>
-                    <exclusion>
-                        <groupId>org.slf4j</groupId>
-                        <artifactId>slf4j-api</artifactId>
-                    </exclusion>
-                    <exclusion>
-                        <groupId>org.slf4j</groupId>
-                        <artifactId>slf4j-log4j12</artifactId>
-                    </exclusion>
-                    <exclusion>
-                        <groupId>log4j</groupId>
-                        <artifactId>log4j</artifactId>
-                    </exclusion>
-                </exclusions>
-            </dependency>
-            <dependency>
-                <groupId>org.apache.tika</groupId>
-                <artifactId>tika-core</artifactId>
-                <version>1.17</version>
-            </dependency>
```

Comment:
We want to leave this here so that the version is declared globally. We use the parent pom this way to a) have a singular place for versioning and b) so that if anyone wants a dependency they automatically get the version we support